### PR TITLE
Solution & Test for #13

### PR DESCRIPTION
**Pull request by seemcat**

### CODE REVIEW

File: solutions/13.js [hunk 1]
```diff
@@ -13,6 +13,24 @@ const solution = (str) => {
...
+const solution1 = (string) => {
+  let indexLength = string.length - 1;
+  let reverseString = '';
+
+  for(i = 0; i <= indexLength; i++){
+    reverseString = string[i] + reverseString;
+  }
+  if(string === reverseString){
+    return true;
+  } else {
```
Comment (dsope05) [inline solutions/13.js]: you don't need this else statement. can be removed.

File: test/13.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const expect = require('chai').expect;
 let solution = require('../solutions/13').solution;
+let solution1 = require('../solutions/13').solution;
```
Comment (dsope05) [inline test/13.js]: remove your code from 13.js and import solutions1 from 13a.js

File: solutions/13.js (hunk 1 above)
Comment (seemcat) [inline solutions/13.js]: Resolved, thanks.

File: test/13.js (hunk 2 above)
Comment (seemcat) [inline test/13.js]: Resolved, thanks.

Comment (seemcat) [inline test/13.js]: How do I merge?

File: solutions/13.js [hunk 3]
```diff
@@ -13,6 +12,22 @@ const solution = (str) => {
 	return true;
 };
 
+// Maricris Bonzo: @seemcat
+const solution1 = (string) => {
+  let indexLength = string.length - 1;
+  let reverseString = '';
+
+  for(i = 0; i <= indexLength; i++){
+    reverseString = string[i] + reverseString;
```
Comment (yjlim5) [inline solutions/13.js]: This does not look like it will reverse the string.  Can you `console.log(reverseString)` to make sure?  and run `npm run test`.

Comment (seemcat) [inline solutions/13.js]: It passes! 
screen shot 2017-06-15 at 11 21 41 am

Comment (seemcat) [inline solutions/13.js]: That is me console.logging the reverseString.

**Comment by yjlim5** [comment]
Thanks for double checking. This code is approved and merged through PR #104.

**Pull request closed without merging**